API/Controllers/OPDSController.cs

```diff
@@ -107,10 +152,7 @@ public OpdsController(IUnitOfWork unitOfWork, IDownloadService downloadService,
     [Produces("application/xml")]
     public async Task<IActionResult> Get(string apiKey)
     {
-        var userId = await GetUser(apiKey);
-        if (!(await _unitOfWork.SettingsRepository.GetSettingsDtoAsync()).EnableOpds)
-            return BadRequest(await _localizationService.Translate(userId, "opds-disabled"));
-
+        var userId = (int)HttpContext.Items[UserId]!;
```

Comment:
Refactor this into a method or a getter so that we aren't duplicating code. I also like a space after the cast.